paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -288,7 +281,7 @@ class Paparazzi @JvmOverloads constructor(
           // CompositionContext, which requires first finding the "content view", then using that
           // to find a root view with a ViewTreeLifecycleOwner
           viewGroup.id = android.R.id.content
-          PaparazziComposeOwner.register(viewGroup)
+          PaparazziViewOwners.register(viewGroup)
```

Comment:
should this be moved out of this block to support all views, as suggested by the commit message?

Reply:
Whoops thanks for catching! I lost that change in a git undo.